rmw_fastrtps_cpp/package.xml

```diff
@@ -7,10 +7,10 @@
   <maintainer email="ricardogonzalez@eprosima.com">Ricardo González</maintainer>
   <license>Apache License 2.0</license>
 
-  <buildtool_depend>ament_cmake</buildtool_depend>
+  <buildtool_depend>ament_cmake_ros</buildtool_depend>
   <buildtool_depend>fastrtps_cmake_module</buildtool_depend>
 
-  <buildtool_export_depend>ament_cmake</buildtool_export_depend>
+  <buildtool_export_depend>ament_cmake_ros</buildtool_export_depend>
```

Comment:
Sorry, I forgot about this. I've reverted that change and rebased against master. Thanks!